scripts/apidoc_full.sh

```diff
@@ -19,7 +19,7 @@ fi
 CONFIG_BAK=${CONFIG_H}.bak
 cp -p $CONFIG_H $CONFIG_BAK
 
-scripts/config.pl realfull
+scripts/config.pl docfull
```

Comment:
Since we only use `realfull` to generate the documentation (which was its original purpose in 1989caf71c1d453c26f150a412af75795b35a980), we don't document it, and the library doesn't even build (`check_config.h` would signal conflicts), I think we should change what `realfull` does to be useful for its intended purpose and not introduce a new name.